### PR TITLE
Feature/post model

### PR DESCRIPTION
**Pull request by damian-pagowski**

### CODE REVIEW

File: api-server/routes/posts.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      const { category } = req.params
+      console.log(category)
+      Post.find({ category })
+        .then(data => res.json(data))
+        .catch(error => res.status(400).json(error))
+    })
+    .post((req, res) => {
+      const { category } = req.params
+      new Post({ ...req.body, category })
+        .save()
```
Comment (dastrong) [inline api-server/routes/posts.js]: Idk if this save method is required. 

I believe you only need to call that after you've done manipulated (updated) something within.

Comment (damian-pagowski) [reply api-server/routes/posts.js]: so how do you want persist entity in database without calling this save() method?

Comment (dastrong) [reply api-server/routes/posts.js]: Using the `new Post` or `Post.create` methods automatically save it to the database.

For example: 
If you wanted to add that postID to a user's document, you would create the post like above, find the user, add it, and then save the user.

Comment (damian-pagowski) [reply api-server/routes/posts.js]: about MyModel.create(docs):

> Shortcut for saving one or more documents to the database. MyModel.create(docs) does new MyModel(doc).save() for every doc in docs.

https://mongoosejs.com/docs/api.html#model_Model.create

**Pull request merged**